soc/arm/silabs_exx32/efr32bg22/Kconfig.defconfig.efr32bg22

```diff
@@ -0,0 +1,24 @@
+# Silicon Labs EFR32BG22 (Blue Gecko) MCU configuration options
+
+# Copyright (c) 2021 Sateesh Kotapati
+# SPDX-License-Identifier: Apache-2.0
+
+config GPIO_GECKO
+	default y
+	depends on GPIO
+
+config I2C_GECKO
+	default y
+	depends on I2C
+
+config SOC_FLASH_GECKO
+	default y
+	depends on FLASH
+
+config SPI_GECKO
+	default y
+	depends on SPI
+	
+config COUNTER_GECKO_STIMER	
+	default y
```

Comment:
Is this driver mandatory for the efr32bg22? With this line it is always compiled in (so if PM is enabled and if it is disabled).

Reply:
This driver is required even if PM is not enabled.